Makefile

```diff
@@ -1,13 +1,13 @@
-clean:
-	# Remove the build
-	rm -rf url.egg-info build dist
-	# And all of our pyc files
-	find . -name '*.pyc' | xargs -n 100 rm
-	# And lastly, .coverage files
-	find . -name .coverage | xargs rm
+.PHONY: test
+test:
+	nosetests --with-coverage
```

Comment:
We've got `rednose` and other options turned on in `setup.cfg`, but `--with-timer` is a good idea.